components/calendar/data/data/mod.rs

```diff
@@ -1,37 +1,18 @@
 // @generated
 include!("macros.rs");
-/// Implement `DataProvider<M>` on the given struct using the data
```

Comment:
Question: where did these docs go

Reply:
I decided these should not be public, only locally usable.

Reply:
Why did you decide that? I thought we had decided that these macros were public API of the data crates. We just talked about that together earlier this week.

Reply:
They haven't been public before, and `impl_data_provider` doesn't work reexported because it cannot address the individual macros (well, it can, but then it wouldn't work *locally* anymore). AnyProvider can be implemented through the impl_dynamic_data_provider macro.

Reply:
Oh I see, this is `impl_data_provider`. But shouldn't `impl_any_provider` still work reexported? It doesn't reference macros, only types. `impl_dynamic_data_provider` needs a list of keys.

Reply:
ok